nototools/unittests/font_tests.py

```diff
@@ -330,25 +330,55 @@ def test_format_4_cmap_preset(self):
 
 
 class TestLigatures(FontTest):
-    """Tests formation or lack of formation of ligatures."""
+    """Tests formation or lack of formation of ligatures.
+
+    Must have the following attributes:
+        active: List of (parameters, (sequence, sequence, ...)) tuples, for
+            which each sequence should form a ligature when shaped with the
+            given parameters. The ligature should not be formed by default if
+            parameters are provided.
+        inactive: Similar list of tuples, for sequences which should form a
+            ligature under inverse circumstances.
```

Comment:
Given the same arguments, the inactive case fails where the active case succeeds, and vice-versa.  I think you just say something like that.  That really is what is going on-- there's no 'inverse' or 'reverse' circumstances.